styles/etools-table-styles.js

```diff
@@ -138,7 +138,7 @@ export const etoolsTableStyles = css`
       padding-left: 5px !important;
       padding-right: 5px !important;
       width: 20px;
-      color: var(--primary-color);
+      color: var(--expand-cell-color, #000000);

```

Comment:
if --expand-cell-color is a newly introduced css var , in the other apps where this variable is not initialised it will fallback to  #000000 which is probably not desired.
```suggestion
      color: var(--expand-cell-color, --primary-color);

```
Also add it to the README file